{"name": "Проверка создания дубля курьера", "status": "failed", "statusDetails": {"message": "AssertionError: assert 'Этот логин у...буйте другой.' == 'Этот логин уже используется'\n  - Этот логин уже используется\n  + Этот логин уже используется. Попробуйте другой.", "trace": "self = <test_reg_double_courier_api.TestDoubleCourierApi object at 0x00000211FB6F5A48>\ncreate_new_courier_and_del = {'firstName': 'saske', 'login': 'jmubpwg', 'password': 'oelcqiz'}\n\n    @allure.title('Проверка создания дубля курьера')\n    def test_double_courier_api(self, create_new_courier_and_del):\n        payload = create_new_courier_and_del\n        requests.post(Constants.URL + Constants.END_COURIER, data=payload)\n        response_double = requests.post(Constants.URL + Constants.END_COURIER, data=payload)\n        r_double = response_double.json()\n        assert response_double.status_code == 409\n>       assert r_double['message'] == Data.error_same_courier\nE       AssertionError: assert 'Этот логин у...буйте другой.' == 'Этот логин уже используется'\nE         - Этот логин уже используется\nE         + Этот логин уже используется. Попробуйте другой.\n\ntests\\test_reg_double_courier_api.py:15: AssertionError"}, "start": 1711317863338, "stop": 1711317864104, "uuid": "10fff0c1-24bb-47c4-9967-fa34208c229c", "historyId": "290c23b57010c2d5a414b9b247f4dca9", "testCaseId": "290c23b57010c2d5a414b9b247f4dca9", "fullName": "tests.test_reg_double_courier_api.TestDoubleCourierApi#test_double_courier_api", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_reg_double_courier_api"}, {"name": "subSuite", "value": "TestDoubleCourierApi"}, {"name": "host", "value": "plrx-00015313"}, {"name": "thread", "value": "7756-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_reg_double_courier_api"}]}